components/PageHeader.vue

```diff
@@ -167,7 +175,6 @@
     left: 0;
     z-index: 1030;
     padding: 0;
-    border-bottom: 1px solid $whitegrey;
```

Comment:
Not sure what this border used to do, but just checking if we should really get rid of it?

Reply:
Yes, it was recommended by Romain. There will be a box-shadow instead when merged in with master.